src/main/java/uk/gov/beis/els/categories/televisions/model/validation/TelevisionsFormSequenceProvider.java

```diff
@@ -4,24 +4,16 @@
 import java.util.List;
```

Comment:
PR already done for this file in [ELG-6](https://github.com/UKGovernmentBEIS/energy-label-service/pull/87)